frontend-plugin-core/src/main/java/com/github/eirslett/maven/plugins/frontend/lib/YarnRunner.java

```diff
@@ -59,6 +60,16 @@ private static List<String> buildArguments(final YarnExecutorConfig config, Prox
     }
 
     /**
+     * <p>
+     * Running {@code yarn node --version} in yarn classic gives us an output like this:
```

Comment:
Could we run `yarn --versions` and/or `node --version` ?

Both seem to produce just the version string with a newline at the end for me at least so maybe there would be no need for parsing at all?

Reply:
`yarn --versions` doesn't work in berry.

`node --version` won't necessarily give us the correct node version, it can end up using the version manager instead of the frontend-maven-plugin so the information would be wrong.

I would've preferred it if it was possible to be simple.